tests/extra_test.py

```diff
@@ -46,43 +64,21 @@ def parsed(self, ctx):
                         # Check if description matches the desired format
                         if description.strip() == "Usage: on='{on}'":
                             parameter["name"] = "on"
-
-        # Drop requirement for @odata.type since it's not actually required
-        for schema in ctx.document["components"]["schemas"].values():
-            if "required" in schema:
-                schema["required"] = [i for i in schema["required"] if i != "@odata.type"]
-                if not schema["required"]:
-                    del schema["required"]
+                        if "content" in parameter.keys():
+                            parameter["schema"] = parameter["content"].get("application/json", {}).get("schema", {})
+                            del parameter["content"]
+        # Drop requirement for @odata.type since it's not actually enforced
+        for schema in ctx.document.get("components", {}).get("schemas", {}).values():
             if isinstance(schema, dict):
+                self._drop_required(schema, "@odata.type")
                 for s in schema.get("allOf", []):
-                    if "required" in s:
-                        s["required"] = [i for i in s["required"] if i != "@odata.type"]
-                        if not s["required"]:
-                            del s["required"]
-
-        ctx.document.setdefault("security", []).append({"token": []})
-        ctx.document.setdefault("components", {}).setdefault("securitySchemes", {}).setdefault(
-            "token",
-            {"type": "http", "scheme": "bearer", "bearerFormat": "JWT"},
-        )
-
-        # Rebuild Tags
-        ctx.document["tags"] = [
-            {"name": tag}
-            for tag in set(
-                tag
-                for operations in ctx.document.get("paths", {}).values()
-                for details in operations.values()
-                if isinstance(details, dict)
-                for tag in details.get("tags", [])
-            )
-        ]
-
+                    self._drop_required(s, "@odata.type")
         return ctx
 
 
 class MSGraphCulled(MSGraph, Cull):
-    pass
+    def paths(self, ctx: "Init.Context") -> "Init.Context":
+        return ctx
```

Comment:
The Cull/Reduce paths() would set ctx.paths non and limit the initialization of schemas to schemas identified as required by tracking the schemas required for the operations.

Why avoid this?

Reply:
I forget specifically what I was testing for there, but I think I was investigating something to do with schemas being ignored when they shouldn't be. 

ADHD moment... It can be set back to `pass` for now, and I'll jump back into that investigation later.

Reply:
remove - so the super() can take care.
tracking failing is a bug we'll address properly, if you find the schema/selectors required to reproduce.